Incorrect properties file in META-INF

When adding the plugin as a dependency in buildSrc (it is applied in a convention plugin), the build fails with the following error:

```
> Task :buildSrc:generateExternalPluginSpecBuilders FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':buildSrc:generateExternalPluginSpecBuilders'.
> implementationClass must not be null
```

It can be reproduced with a setup similar to this:
```
plugins {
    `kotlin-dsl`
}

repositories {
    mavenCentral()
}

dependencies {
    implementation("io.github.luca992.multiplatform-swiftpackage:io.github.luca992.multiplatform-swiftpackage.gradle.plugin:2.1.2")
}
```

I am pretty sure the reason for this error is, that there is an incorrect properties file in META-INF coming directly from the [source](https://github.com/luca992/multiplatform-swiftpackage/blob/master/src/main/resources/META-INF/gradle-plugins/com.chromaticnoise.multiplatform-swiftpackage.properties) with an incorrect spelling of `implementationClass` (should be `implementation-class`). As the implementation class is also set in `build.gradle` and META-INF contains a second correct properties file in the resulting jar, the incorrect file most likely can be deleted.

--- a/src/main/resources/META-INF/gradle-plugins/com.chromaticnoise.multiplatform-swiftpackage.properties
+++ b/src/main/resources/META-INF/gradle-plugins/com.chromaticnoise.multiplatform-swiftpackage.properties
@@ -1,1 +1,0 @@
-implementationClass=com.chromaticnoise.multiplatformswiftpackage.MultiplatformSwiftPackagePlugin

